src/classes/CRLP_Batch_Base_Skew.cls

```diff
@@ -165,7 +165,8 @@ public abstract class CRLP_Batch_Base_Skew extends CRLP_Batch_Base {
                             this.statefulCacheOfRollupsToBeProcessed);
                 } else if (this.jobType == CRLP_RollupProcessingOptions.RollupType.ContactSoftCredit) {
                     addlFilter = CRLP_Query_SEL.buildCommonQueryFilterOnOpportunity(Contact.SObjectType, Partial_Soft_Credit__c.SObjectType);
-                } else if (this.jobType == CRLP_RollupProcessingOptions.RollupType.AccountContactSoftCredit) {
+                } else if (this.jobType == CRLP_RollupProcessingOptions.RollupType.AccountContactSoftCredit || 
```

Comment:
This also needs to be removed because the code in the if block is extremely specific Accounts, not Contacts. Line 170 below (the `addFilter=` line) shows that it's specific to Accounts).